uni/lib/model/entities/bug_report.dart

```diff
@@ -1,18 +1,40 @@
 /// Stores information about Bug Report
+import 'package:json_annotation/json_annotation.dart';
 import 'package:tuple/tuple.dart';
 
+part 'bug_report.g.dart';
+
+class TupleConverter
+    extends JsonConverter<Tuple2<String, String>?, Map<String, dynamic>?> {
+  const TupleConverter();
+
+  @override
+  Tuple2<String, String>? fromJson(Map<String, dynamic>? json) {
+    if (json == null) return null;
+    return Tuple2<String, String>(
+        json['item1'] as String, json['item2'] as String);
+  }
+
+  @override
+  Map<String, dynamic>? toJson(Tuple2<String, String>? object) {
+    if (object == null) return null;
+    return {
+      'item1': object.item1,
+      'item2': object.item2,
+    };
```

Comment:
The original `toMap` used only `item2` from `bugLabel`. Won't this break that logic?

Reply:
```suggestion
    return {
      'item2': object.item2,
    };
```

Reply:
If it only used the second item, why is it even a tuple?

Reply:
Data classes have meaning inside the app, e.g. in widgets. It is not uncommon that some properties that the app needs are not serialized